demos/wascc/fileserver/k8s.yaml

```diff
@@ -11,7 +11,6 @@ spec:
       name: fileserver-wascc
       env:
```

Comment:
Bad communication on my part. This whole `env` section should be removed as we no longer need anything in it